apps/aeoracle/src/aeo_extend_tx.erl

```diff
@@ -98,10 +102,13 @@ signers(#oracle_extend_tx{oracle = OraclePK}, _) ->
     {ok, [OraclePK]}.
 
 -spec process(tx(), aetx:tx_context(), aec_trees:trees(), height(), non_neg_integer()) -> {ok, aec_trees:trees()}.
-process(#oracle_extend_tx{oracle = OraclePK, nonce = Nonce, fee = Fee, ttl = TTL},
+process(#oracle_extend_tx{oracle = OraclePKOrName, nonce = Nonce, fee = Fee, ttl = TTL},
         _Context, Trees0, Height, _ConsensusVersion) ->
     AccountsTree0 = aec_trees:accounts(Trees0),
     OraclesTree0  = aec_trees:oracles(Trees0),
+    NamesTree0 = aec_trees:ns(Trees0),
+
+    {ok, OraclePK} = aens:resolve_decoded(oracle_pubkey, OraclePKOrName, NamesTree0),
```

Comment:
This affects the originator of the tx, that is responsible for signing and whose nonce is affected. This seems too involved.

* As both signature and nonce are closely coupled with tx originator, why would originator use a name? I may be missing the use case.
* If mempool checked tx before accepting it in pool (assuming it does - did not check), how could it reliably check that signature is from correct tx originator?
* If attempting some sort of ordering of tx for the originatorpubkey-nonce contraints, this makes it impossible to reason on this contraints distinctly from creation of block candidate. (That could be ok - not sure.) cc @gorillainduction

Reply:
I agree @lucafavatella the case of name resolvement for extend tx can be problematic and is not beneficial to user experience, I propose to remove name resolvement here.